### DISCUSSION TITLE
Coolify Traffic

### DISCUSSION THREAD

**Discussion by MentalGear** [coollabsio/coolify#4242] [Q&A]
The docs are great, easy to read and not bulky - great job on making a PaaS much more accessible. One thing I'm not yet clear about is if whether any visitor traffic is ever being routed through Coolify?
Or is Coolify just for managing the underlying servers/apps?

For example, when using Coolify Cloud (US?) with a remote app server in the EU and using Coolify's load balancer, does visitor traffic gets routed first through Coolify Cloud before reaching the actual server ?

**Comment by oshimayoan** [comment]
Yes, this is also what I'm wondering. If every traffic goes through Coolify first, isn't it gonna make the instance where our Coolify lives heavier and need more resources based on how big the traffic to the remote instance is?

Could anyone clarify this?
Here is the only source that I can found about: https://docs.coollabs.io/coolify/v3/architecture#http-proxy

Thanks in advance
